airflow/providers/postgres/example_dags/example_postgres.py

```diff
@@ -70,8 +70,9 @@
             BETWEEN SYMMETRIC DATE '{{ params.begin_date }}' AND DATE '{{ params.end_date }}';
             """,
         params={'begin_date': '2020-01-01', 'end_date': '2020-12-31'},
```

Comment:
This example is wrong (not your change, but oops)

```suggestion
        parameters={'begin_date': '2020-01-01', 'end_date': '2020-12-31'},
```

Reply:
I don't think it's wrong. The code references to params by `params.begin_date`,` params.end_date`
`parameters` is to pass the assignment to sqlalchemy engine (which means you will not see the full rendered query in render tab)

Reply:
Oh yeah, Better would be to use parameters so that we don't risk SQLI:

```
            BETWEEN SYMMETRIC DATE :begin_date: AND DATE :end_date:';
            """,
        parameters={'begin_date': '2020-01-01', 'end_date': '2020-12-31'},
```
(untested)

Reply:
agree.
@victorphoenix3 can you take care of it?
I think this is also the first example in all of our docs that will show the usage of `parameters`